plugins/org.eclipse.sirius.tests.swtbot.support/src/org/eclipse/sirius/tests/swtbot/support/api/AbstractSiriusSwtBotGefTestCase.java

```diff
@@ -1933,6 +1933,11 @@ protected void tearDown() throws Exception {
         // org.eclipse.ui.internal.statushandlers.StatusHandlerRegistry.<init>
         // on close.
         ResourcesPlugin.getWorkspace().save(true, null);
+        
+        if (editor != null) {
```

Comment:
In theory, it is not possible to be here with `editor != null` because it is set to null with the above `new TestCaseCleaner(this).clearAllFields();`. Do you have steps to reproduce?